bin/circleci/setup-integration-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# TODO: Update this occasionally from here:
```

Comment:
We've also got an issue open to update dependencies, since we're pretty far behind now and a mass upgrade will inevitably break things :(